flake.nix

```diff
@@ -0,0 +1,41 @@
+{
+  description = "Hardware and Firmware of the Solo Quadruped Master Board.";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+  };
+
+  outputs =
+    { self, nixpkgs }:
+    let
+      pkgs = nixpkgs.legacyPackages.x86_64-linux;
+    in
+    {
+      packages.x86_64-linux.default = pkgs.stdenv.mkDerivation {
+
+        pname = "odri-masterboard-sdk";
```

Comment:
@olivier-stasse : if I remember correctly, you had to adapt the name for ROS, but I don't see that in package.xml. 
Can you recall us what you used ? I think it would be better to unify this

Reply:
found: https://index.ros.org/p/odri_master_board_sdk